aptos-move/framework/aptos-framework/sources/vesting.move

```diff
@@ -1375,35 +1515,54 @@ module aptos_framework::vesting {
         let vested_amount = fraction(GRANT_AMOUNT, 3, 48);
         let remaining_grant = GRANT_AMOUNT - vested_amount;
         let pending_distribution = rewards + vested_amount;
-        assert!(remaining_grant(contract_address) == remaining_grant, remaining_grant(contract_address));
-        stake::assert_stake_pool(stake_pool_address, remaining_grant, 0, 0, pending_distribution);
+        assert!(
+            remaining_grant(contract_address) == remaining_grant,
+            remaining_grant(contract_address),
+        );
+        stake::assert_stake_pool(
+            stake_pool_address, remaining_grant, 0, 0, pending_distribution
+        );
 
         // Fast forward to the end of the fourth period. We can call vest() 3 times to vest the last 3 periods.
         timestamp::fast_forward_seconds(VESTING_PERIOD * 3);
         vest(contract_address);
         vested_amount = fraction(GRANT_AMOUNT, 2, 48);
         remaining_grant = remaining_grant - vested_amount;
         pending_distribution = pending_distribution + vested_amount;
-        stake::assert_stake_pool(stake_pool_address, remaining_grant, 0, 0, pending_distribution);
+        stake::assert_stake_pool(
+            stake_pool_address, remaining_grant, 0, 0, pending_distribution
+        );
         vest(contract_address);
         vested_amount = fraction(GRANT_AMOUNT, 1, 48);
         remaining_grant = remaining_grant - vested_amount;
         pending_distribution = pending_distribution + vested_amount;
-        stake::assert_stake_pool(stake_pool_address, remaining_grant, 0, 0, pending_distribution);
+        stake::assert_stake_pool(
+            stake_pool_address, remaining_grant, 0, 0, pending_distribution
+        );
         // The last vesting fraction (1/48) is repeated beyond the first 3 periods.
         vest(contract_address);
         remaining_grant = remaining_grant - vested_amount;
         pending_distribution = pending_distribution + vested_amount;
-        stake::assert_stake_pool(stake_pool_address, remaining_grant, 0, 0, pending_distribution);
+        stake::assert_stake_pool(
+            stake_pool_address, remaining_grant, 0, 0, pending_distribution
+        );
         assert!(remaining_grant(contract_address) == remaining_grant, 0);
 
         stake::end_epoch();
         let total_active = with_rewards(remaining_grant);
         pending_distribution = with_rewards(pending_distribution);
         distribute(contract_address);
         stake::assert_stake_pool(stake_pool_address, total_active, 0, 0, 0);
-        assert!(coin::balance<AptosCoin>(shareholder_1_address) == shareholder_1_bal + pending_distribution / 4, 0);
-        assert!(coin::balance<AptosCoin>(shareholder_2_address) == shareholder_2_bal + pending_distribution * 3 / 4, 1);
+        assert!(
+            coin::balance<AptosCoin>(shareholder_1_address) == shareholder_1_bal + pending_distribution
```

Comment:
can we create an issue to track this unideal line breaking?

Reply:
yeah